lang/ast/structs.go

```diff
@@ -4617,15 +4613,13 @@ func (obj *StmtInclude) Init(data *interfaces.Data) error {
 // generally increases the size of the AST when it is used. It calls Interpolate
 // on any child elements and builds the new node with those new node contents.
 func (obj *StmtInclude) Interpolate() (interfaces.Stmt, error) {
-	args := []interfaces.Expr{}
-	if obj.Args != nil {
-		for _, x := range obj.Args {
-			interpolated, err := x.Interpolate()
-			if err != nil {
-				return nil, err
-			}
-			args = append(args, interpolated)
+	args := make([]interfaces.Expr, 0, len(obj.Args))
```

Comment:
Please leave this definition site as it was, the compiler should be able to optimize this for us. If not, it's okay, it's not an area we need to optimize at this time. It's much harder (at least for me) to read.